client/src/ui/components/worldmap/armies/ArmyInfoLabel.tsx

```diff
@@ -42,10 +43,11 @@ const RaiderInfo = ({ army }: ArmyInfoLabelProps) => {
     account: { account },
   } = useDojo();
 
+  const updatedArmy = useArmyByArmyEntityId(BigInt(army.entity_id));
```

Comment:
Do we really need to use another `useEntityQuery` here? We already have the army passed as props